tests/test_cases/chdir_getcwd.c

```diff
@@ -0,0 +1,25 @@
+#include <unistd.h>
+#include <stdio.h>
+
+int main() {
+  int buffersize = 256;
+  char cwd[buffersize];
+
+  char* result = getcwd(cwd, sizeof(cwd));
+
+  if (result == NULL)
+    perror("getcwd() error");
+  else
+    printf("current working directory is: %s :: %s\n", cwd, result);
+  //---------------------------
+  chdir("/test");
+  //---------------------------

```

Comment:
Instead of adding two comment lines; `chdir (...);  //Working directory` style could be a bit better, doesn't affect the functionality though.